server/src/lib/api.ts

```diff
@@ -101,41 +105,44 @@ export default class API {
       return Promise.resolve(this.sentencesCache);
     }
 
-    return this.getFilesInFolder(this.getSentenceFolder())
-      .then((files: string[]) => {
-        return Promise.all(files.map(filename => {
-
-          // Only parse the top-level text files, not any sub folders.
-          if (filename.split('.').pop() !== 'txt') {
-            return null;
-          }
-
-          let filepath = path.join(this.getSentenceFolder(), filename);
-          return this.getFileContents(filepath);
-        }));
-      })
-
-
-      // Chop the array of content strings into an array of sentences.
-      .then((values: string[]) => {
-        let sentences: string[] = [];
-        let sentenceArrays = values.map(fileContents => {
-          if (!fileContents) {
-            return [];
-          }
-
-          // Remove any blank line sentences.
-          let fileSentences = fileContents.split('\n');
-          return fileSentences.filter(sentence => { return !!sentence; });
-        });
-
-        sentences = sentences.concat.apply(sentences, sentenceArrays);
-        console.log('sentences found', sentences.length);
-        this.sentencesCache = sentences;
-      })
-      .catch((err: any) => {
-        console.error('could not retrieve sentences', err);
-      });
+    return (
```

Comment:
another curious case of additional parens

Reply:
yup, this is a Prettier convention that I couldn't find a config for. i wouldn't write code like this, but it's not a deal-breaker either